vite.config.ts

```diff
@@ -62,6 +62,9 @@ export default defineConfig({
 	define: {
 		'process.env.NODE_ENV': '"production"',
 	},
+	worker: {
+		format: 'es',
```

Comment:
we are also using a worker to search the sveltekit documentation and the es worker format is not supported by firefox...i don't think there's a way to have them built in different formats right?